airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/NormalizationSummaryCheckActivityImpl.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.sync;
+
+import io.airbyte.persistence.job.JobPersistence;
+import io.airbyte.persistence.job.models.AttemptNormalizationStatus;
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Singleton
+@AllArgsConstructor
+@NoArgsConstructor
+@Slf4j
+public class NormalizationSummaryCheckActivityImpl implements NormalizationSummaryCheckActivity {
+
+  @Inject
+  private JobPersistence jobPersistence;
```

Comment:
Per both my preference and [Micronaut's preference](https://guides.micronaut.io/latest/micronaut-dependency-injection-types-gradle-java.html)
> Field injection makes it harder to understand a class’s requirements, making it easier to get a NullPointerException when testing a class using Field Injection. We recommend you use [Constructor Injection](https://guides.micronaut.io/latest/micronaut-dependency-injection-types-gradle-java.html#benefits-of-constructor-injection).

could we change this to be constructor injected?

Reply:
Also, the `@AllArgsConstructor` will create a constructor that wins over the field injection.  This is probably working because we don't have any qualifiers or configuration properties here.  However, per @colesnodgrass's request, we should use explicit constructors here and avoid Lombok/field injection to make the code easier to read.